src/internals/legend.js

```diff
@@ -312,7 +312,7 @@ extend(ChartInternal.prototype, {
 		const isTouch = $$.inputType === "touch";
 
 		item
-			.attr("class", id => $$.generateClass(CLASS.legendItem, id))
+			.attr("class", id => `${!!item.attr("class") && item.attr("class")}${$$.generateClass(CLASS.legendItem, id)}`)
```

Comment:
Hi @netil , I read your comment. Thanks for considering.  
But I found a bug on my code and fixed it.  
This line shoud be 
```
		item
			.attr("class", id => `${this.className ? this.className : ""}${$$.generateClass(CLASS.legendItem, id)}`)
```
It's because I misunderstood how d3 selection working.  
So it should be changed like this that get each item's className.

Reply:
I didn't merged exactly as you wrote.
You can check the changed code on below link:
- https://github.com/naver/billboard.js/commit/46ebf35e2fa5b4bdfcff61638eec8312d5156ef0

Reply:
Better than this! :) Thanks for adding me as a contributor.